recipes/python-lzo/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+export CPPFLAGS="-I$PREFIX/include -I$PREFIX/include/lzo $CPPFLAGS"
+export LDFLAGS="-L$PREFIX/lib $LDFLAGS"
+
+$PYTHON setup.py install
```

Comment:
Not sure, sorry. Presumably @jd-boyd can confirm, and I am happy to update the package.